rocketpy/Motor.py

```diff
@@ -115,6 +115,7 @@ def __init__(
         self,
         thrustSource,
         burnOut,
+        distanceNozzlePropellant,
```

Comment:
Please add this `distanceNozzlePropellant`information on the info() and allInfo() methods